apps/outlook/src/pages/taskpane/index.ts

```diff
@@ -1,2 +0,0 @@
-import Taskpane from "./taskpane";
```

Comment:
Interesting that you moved that into the `features/router` folder. Any specific purpose?

I probably see a route as a type and not a feature in itself, e.g. the feature could be the taskpane and it will include a route, page component, some utility logic etc. What's your take?

Reply:
I just mimicked the structure of apps/extension to keep it uniform. We could have our routes in src/pages for all 3 apps maybe?

Reply:
Yes, that actually makes sense. Follow nextjs' convention to have it consistent everywhere. 👍